bolt-sidecar/src/driver.rs

```diff
@@ -105,7 +105,7 @@ impl<C: StateFetcher, BLS: SignerBLS, ECDSA: SignerECDSA> SidecarDriver<C, BLS,
         commitment_signer: ECDSA,
         fetcher: C,
     ) -> eyre::Result<Self> {
-        let mevboost_client = MevBoostClient::new(cfg.mevboost_url.clone());
+        let mevboost_client = ConstraintClient::new(cfg.mevboost_url.clone());
```

Comment:
Change `mevboost_url` to `constraints_url`

Reply:
I felt it a bit confusing (cc @merklefruit wdyt? )

Reply:
`constraints_url` is more idiomatic considering mev-boost is "implementation" rather than spec.
Just like in the CL you have to specify `--builder` to point to mev-boost because it refers to the builder API.

Reply:
ref https://lighthouse-book.sigmaprime.io/builders.html#how-to-connect-to-a-builder